common/src/app/common/path/shapes_to_path.cljc

```diff
@@ -4,22 +4,52 @@
 ;;
 ;; Copyright (c) UXBOX Labs SL
 
-(ns app.util.path.shapes-to-path
+(ns app.common.path.shapes-to-path
   (:require
    [app.common.data :as d]
    [app.common.geom.matrix :as gmt]
    [app.common.geom.point :as gpt]
-   [app.common.geom.shapes :as gsh]
+   [app.common.geom.shapes.common :as gsc]
    [app.common.geom.shapes.path :as gsp]
-   [app.util.path.commands :as pc]))
-
-(def bezier-circle-c 0.551915024494)
-(def dissoc-attrs [:x :y :width :height
-                   :rx :ry :r1 :r2 :r3 :r4
-                   :medata])
-(def allowed-transform-types #{:rect
-                               :circle
-                               :image})
+   [app.common.path.bool :as pb]
+   [app.common.path.commands :as pc]))
+
+(def ^:const bezier-circle-c 0.551915024494)
+
+(def ^:const dissoc-attrs
```

Comment:
I don't think is a good idea declare **const** for not primitive JS values. I mean, it is OK, but the compiler will use the const values as inline values, and if this vars are used more than one time... it will increase the size of the final bundle without any real additional benefit.